user_interaction/static/themes/auto.css

```diff
@@ -0,0 +1,2 @@
+@import url(./standard-theme.css);
```

Comment:
I didn't succeed into adding a media query to this that also worked on Internet Explorer or other older browsers.

Reply:
I don't think we need to support old browsers like IE, as I doubt our members would still be using these

Reply:
Okay, also all browser that don't support `prefers-color-scheme`, which still is 8% according to [caniuse.com](https://caniuse.com/?search=prefers-color-scheme).